R/theme-elements.r

```diff
@@ -226,9 +226,11 @@ element_grob.element_line <- function(element, x = 0:1, y = 0:1,
   default.units = "npc", id.lengths = NULL, ...) {
 
   # The gp settings can override element_gp
-  gp <- gpar(lwd = len0_null(size * .pt), col = colour, lty = linetype, lineend = lineend)
+  gp <- gpar(lwd = len0_null(size * .pt), col = colour, lty = linetype,
+             lineend = lineend, fill = colour)
```

Comment:
Could you use tidyverse-style formatting? The same applies for the other change.

``` r
gp <- gpar(
  lwd = len0_null(size * .pt), col = colour, lty = linetype,
  lineend = lineend, fill = colour
)

Reply:
And maybe reorder the parameters so `col` and `fill` go together and then `lwd`, `lty`, and `lineend`.

Reply:
Sure.